modules/default-requirements-central-services.adoc

```diff
@@ -120,58 +118,52 @@ The StackRox Scanner requires Scanner DB (PostgreSQL 15) to store data. This dat
 [id="default-requirements-central-services-scanner-v4_{context}"]
 == Scanner V4
 
-[discrete]
-== CPU, memory, and storage requirements
-
-[discrete]
-=== Scanner V4 Indexer
+*Scanner V4 Indexer*
 
 The requirements in this table are based on the default of 3 replicas.
 
 |===
 | Scanner V4 Indexer | CPU | Memory
 
 | *Request*
-| 3 cores
-| 4500 MiB
+| 1.5 cores
+| 500 MiB
 
 | *Limit*
-| 6 cores
-| 9 GiB
+| 4 cores
+| 1 GiB
 |===
 
-[discrete]
-=== Scanner V4 Matcher
+*Scanner V4 Matcher*
 
 The requirements in this table are based on the default of 2 replicas.
 
 |===
 | Scanner V4 Matcher | CPU | Memory
 
 | *Request*
-| 2 cores
-| 1000 MiB
+| .5 core
+| 1.5 GiB
 
 | *Limit*
-| 4 cores
-| 4 GiB
+| 1 core
+| 3 GiB
 |===
 
-[discrete]
-=== Scanner V4 DB
+*Scanner V4 DB*
 
 Scanner V4 requires Scanner V4 DB (PostgreSQL 15) to store data. For Scanner V4 DB, a PVC is required to ensure optimal performance. The following table lists the minimum CPU, memory, and storage values required to install and run Scanner V4 DB.
 
 |===
 | Scanner V4 DB | CPU | Memory | Storage
 
 | *Request*
-| 0.2 cores
+| 1 core
 | 3 GiB
-| 50 GiB
+| 4 GiB
 
 | *Limit*
-| 2 cores
+| 4 cores
 | 4 GiB
-| 50 GiB
+| 8 GiB
```

Comment:
I think you're updating the storage limit, which should remain the same (50GiB), instead of the memory (from 4 GiB to 8 GiB).